src/Templates/form.twig

```diff
@@ -37,7 +37,7 @@
 {{ parent() }}
 {% include 'parts/table-footer.twig' %}
 <script type="text/javascript" src="{{baseurl}}js/vendor/jquery.validate.min.js"></script>
-<script type="text/javascript" src="{{baseurl}}js/vendor/tinymce.min.js"></script>
+<script src="//cdn.tinymce.com/4/tinymce.min.js"></script>
```

Comment:
It seemed like a big bunch of stuff to pointlessly store in our repo. Idk, we used to use CDNs for everything, I can't remember why we moved off them